### PR TITLE
add namespace fix for android platform optimizations

### PR DESCRIPTION
**Pull request by headupinclouds**

### CODE REVIEW

File: cmake/projects/ogles_gpgpu/hunter.cmake [hunk 1]
```diff
@@ -15,6 +15,17 @@ hunter_add_version(
...
+    "https://github.com/hunter-packages/ogles_gpgpu/archive/v0.1.6-p1.tar.gz"
+    SHA1
+    b6b2cc03f49dabf0dd72a09b6c760164f6fb6e10
+)
+
+hunter_add_version(
+    PACKAGE_NAME
+    ogles_gpgpu
+    VERSION
+    0.1.6
```
Comment (ruslo) [inline cmake/projects/ogles_gpgpu/hunter.cmake]: Shoud be `0.1.6-p1` I guess.

Comment (headupinclouds) [inline cmake/projects/ogles_gpgpu/hunter.cmake]: Sorry.  Fixed.

**Comment by ruslo** [comment]
Uploading:
* https://ci.appveyor.com/project/ingenue/hunter/build/1.0.1213
* https://travis-ci.org/ingenue/hunter/builds/181228375

**Comment by ruslo** [comment]
Released: https://github.com/ruslo/hunter/releases/tag/v0.16.36

**Pull request merged**